overlord/snapstate/snapstate.go

```diff
@@ -531,6 +531,20 @@ func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup, flags int
 		prev = installHook
 	}
 
+	if snapsup.QuotaGroupName != "" {
```

Comment:
either here or one level up we need to do quota conflict checks

Reply:
Fixed!